libs/monorepo-utils/src/dependencies.test.ts

```diff
@@ -81,8 +81,9 @@ test('findAllMonorepoDependencies yields all dependencies', () => {
   const pkgs = getWorkspacePackageInfo(join(__dirname, '../../..'));
   const basicsPkg = pkgs.get('@votingworks/basics')!;
 
-  // single dependency
+  // simple dependencies
```

Comment:
What exactly do you mean by "simple dependencies"? (To be clear, I don't think I understand "single dependency" either 😅)

Reply:
Yeah I was confused as well. This function finds all of the deps of the given package that are workspace packages (i.e. directories in the monorepo) rather than npm packages. This particular assertion was trying to check a simple case (a package with only one workspace dep), but my changes added another dep, so I updated the comment to try to avoid it being misleading. I don't know that I increased the clarity at all haha